enderio-machines/src/main/java/com/enderio/machines/common/blocks/base/blockentity/MachineBlockEntity.java

```diff
@@ -82,6 +83,9 @@ public abstract class MachineBlockEntity extends EIOBlockEntity
 
     private final boolean supportsActiveState;
 
+    @Nullable
+    private UUID owner;
```

Comment:
Should we be using GameProfile instead?
Also - do we want ownership to be saved to the item when the block is broken? If so this'll need a data component to be put on the item.

Reply:
So we want uuid yes, since we may need to create multiple fakeplayers with that uuid. 
Ownership is imo the last person to place it down, not the first. We can change it, but I don't think it needs to be remembered in the item

Reply:
GameProfile would still work? You can get the UUID from it, and then we have the player's last known name that we could use later for debug/admin info etc.